server/routes/admin.js

```diff
@@ -1,20 +1,80 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 const categoryClient = require('../dbClients/categoriesDB');
 const articleClient = require('../dbClients/articlesDB')
- 
+const ObjectId = require('mongodb').ObjectID;
 
 router.get("/", function (req, res, next) {
   res.send('admin');
 });
 
 router.get('/categories', function (req, res, next) {
   const callback = (error, category) => {
-     res.render("admin-list-categories",{
+    res.render("admin-list-categories", {
       category
-     }) 
+    })
+  }
+  categoryClient.findCategories({},callback);
+});
+
+router.get('/articles', function (req, res, next) {
+  const callback = (error, articles) => {
+    res.render("admin-list-articles", {
+      articles: articles
+    })
+  }
+  articleClient.findArticles({}, callback);
+});
+
+router.get('/articles/edit/:articleId', (req, res) => {
+  let articleId = req.params.articleId;
+  const callback = (error, article) => {
+    res.render("admin-edit-article", {
+      article: article[0]
+    })
+  }
+  articleClient.findArticles({ "_id": ObjectId(articleId) }, callback);
+})
+
+router.post('/articles/edit/:articleId', function (req, res, next) {
+  let articleId = req.params.articleId;
+  const query = req.body;
+  const callback = (error, article) => {
+// we use find articles function to show us list of all articles after update
+    const callbacktwo = (error, articles) => {
```

Comment:
you don't need callbackTwo - just res.redirect to /articles, and that should show the list

 (also callbackTwo is very bad name - call it something like listCallback and editCallback)